src/diffusers/models/cross_attention.py

```diff
@@ -228,11 +232,12 @@ def __call__(self, attn: CrossAttention, hidden_states, encoder_hidden_states=No
         attention_mask = attn.prepare_attention_mask(attention_mask, sequence_length)
 
         query = attn.to_q(hidden_states)
-        query = attn.head_to_batch_dim(query)
 
         encoder_hidden_states = encoder_hidden_states if encoder_hidden_states is not None else hidden_states
         key = attn.to_k(encoder_hidden_states)
         value = attn.to_v(encoder_hidden_states)
+
+        query = attn.head_to_batch_dim(query)
```

Comment:
nit: same statement order as `XFormersCrossAttnProcessor` and m'thinks more readable